flexidate/__init__.py

```diff
@@ -32,7 +31,7 @@ def __init__(self, year=None, month=None, day=None, qualifier=''):
          
     def _cvt(self, val, rjust=2, force=False):
         if val:
-            tmp = unicode(val).strip()
+            tmp = str(val).strip()
```

Comment:
I checked some commands and they worked fine. But I don't think they are backward compatible.